src/molecule/command/base.py

```diff
@@ -108,7 +108,7 @@ def execute_cmdline_scenarios(scenario_name, args, command_args, ansible_args=()
 
         if scenario.config.config["prerun"]:
             LOG.info("Performing prerun...")
-            scenario.config.runtime.prepare_environment()
+            scenario.config.runtime.prepare_environment(install_local=True)
```

Comment:
Cannot see a use case yet but maybe this can com from a flag. Thank you for your fix, I was able to test it locally and it works.